packages/components/date-picker/index.ts

```diff
@@ -13,3 +15,8 @@ export default _DatePicker
 export const ElDatePicker = _DatePicker
 export * from './src/constants'
 export * from './src/props/date-picker'
+
+export type DatePickerInstance = ComponentPublicInstance<
+  DatePickerProps,
+  DatePickerExpose
+>
```

Comment:
This can be put into an `instance.ts` file separately.

Reply:
Updated, PTAL.